packages/legend-application-studio/src/components/editor/edit-panel/GrammarTextEditor.tsx

```diff
@@ -882,17 +887,32 @@ export const GrammarTextEditor = observer(() => {
     const editorModel = editor.getModel();
     if (editorModel) {
       editorModel.updateOptions({ tabSize: TAB_SIZE });
-      if (error?.sourceInformation) {
-        setErrorMarkers(
+      const collectionOfWarnings: {
```

Comment:
this looks like we no longer alert about the error, why can't we separately call `setErrorMarkers` and `setWarningMarkers`?